test/src/main/java/org/apache/accumulo/test/ManagerRepairsDualAssignmentIT.java

```diff
@@ -99,12 +102,21 @@ public void test() throws Exception {
         }
       }
       assertEquals(2, states.size());
+      int deadCount = getZkDeadCount(cluster.getServerContext().getZooReader(),
+          cluster.getServerContext().getInstanceID());
       // Kill a tablet server... we don't care which one... wait for everything to be reassigned
       cluster.killProcess(ServerType.TABLET_SERVER,
           cluster.getProcesses().get(ServerType.TABLET_SERVER).iterator().next());
       Set<TabletMetadata.Location> replStates = new HashSet<>();
       @SuppressWarnings("deprecation")
       TableId repTable = org.apache.accumulo.core.replication.ReplicationTable.ID;
+
+      while (getZkDeadCount(cluster.getServerContext().getZooReader(),
+          cluster.getServerContext().getInstanceID()) <= deadCount) {
```

Comment:
Does this count ever go back down after increasing? If so, this check could be flaky if it went up and then back down before we checked it here.

Reply:
Within the context of the test it should not decrease - whatever would clean the dead server list would only have a narrow window (5 sec).  If it is shown to be a problem, then the check would need to be more complicated.  

If somehow the "same" server was brought back to like, the remainder of the test might have issue too.